src/main/java/com/iceye/esa/snap/dataio/IceyeProductReader.java

```diff
@@ -92,8 +91,10 @@ private static void addIncidenceAnglesSlantRangeTime(final Product product, fina
             final double nearRangeAngle = incidenceAngles[0];
             final double farRangeAngle = incidenceAngles[incidenceAngles.length - 1];
 
-            final double firstRangeTime = netcdfFile.getRootGroup().findVariable(IceyeXConstants.FIRST_PIXEL_TIME).readScalarDouble();
-            final double lastRangeTime = netcdfFile.getRootGroup().findVariable(IceyeXConstants.FIRST_PIXEL_TIME).readScalarDouble() + (netcdfFile.getRootGroup().findVariable(IceyeXConstants.NUM_SAMPLES_PER_LINE).readScalarDouble() / netcdfFile.getRootGroup().findVariable(IceyeXConstants.RANGE_SAMPLING_RATE).readScalarDouble());
+            final double firstRangeTime = netcdfFile.getRootGroup().findVariable(IceyeXConstants.FIRST_PIXEL_TIME).readScalarDouble()*Constants.sTOns;
+            final double samplesPerLine = netcdfFile.getRootGroup().findVariable(IceyeXConstants.NUM_SAMPLES_PER_LINE).readScalarDouble();
+            final double rangeSamplingRate = netcdfFile.getRootGroup().findVariable(IceyeXConstants.RANGE_SAMPLING_RATE).readScalarDouble();
+            final double lastRangeTime = firstRangeTime + samplesPerLine / rangeSamplingRate * Constants.sTOns;
```

Comment:
can't find the  Constants.sTO after rebuild the project at class org.esa.snap.engine_utilities.eo.Constants